app/packs/src/components/models/Inventory.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-underscore-dangle,  camelcase, semi,  no-unused-vars */
+import Element from 'src/models/Element';
+
+export default class Inventory extends Element {
+  static buildEmpty() {
+    return new Inventory({
+      inventory_parameters: [{ cas: null, internal_label: null, purity: null, details: null }]
+    });
+  }
+
+  get inventory_parameters() {
+    return this._inventory_parameters;
+  }
+
+  set inventory_parameters(inventory_parameters) {
+    if (inventory_parameters) {
+      this._inventory_parameters = inventory_parameters;
+    }
+  }
+
+  serialize() {
+    return super.serialize({
+      inventory_parameters: this.inventory_parameters
+    })
+  }
+
+
+  invenParameters(parameter, value) {
```

Comment:
Question: What does this method do? I have seen some invocations but I do not understand its use.

From what I can read it is a convenience writer for the first entry in inventory_parameters. 
I have not read the whole InventoryTab component through so the answer may be hidden there, but I will ask anyway: What is the point of defining inventory_parameters as an array of objects if you only ever use the first object?